packages/rocketchat-i18n/i18n/pt-BR.i18n.json

```diff
@@ -2893,6 +2893,7 @@
   "Send_request_on_offline_messages": "Enviar requisição para mensagens off-line",
   "Send_request_on_visitor_message": "Enviar requisição para mensagens do Visitante",
   "Send_Test": "Enviar teste",
+  "Send_Test_Email": "Enviar email de teste",
```

Comment:
Where did you set the other translation keys? 
There are many new keys defined in `en.i18n.json`, but just one in `pt-BR.i18n.json`.